PCbuild/get_external.py

```diff
@@ -5,6 +5,7 @@
 import pathlib
 import sys
 import time
+import urllib.error
 import zipfile
 from urllib.request import urlretrieve
```

Comment:
```suggestion
import zipfile
from urllib.error import URLError
from urllib.request import urlretrieve
```
Some people may prefer keeping them together?

Reply:
Took a slightly different approach to keep things similar across files.

Reply:
Eventually we should probably have a unified script to download things without the duplication across files, but for now I'd like to keep the scripts similar.